src/ekss/config.py

```diff
@@ -47,6 +47,11 @@ class VaultConfig(BaseSettings):
         " verify the identity of the vault, or True to"
         " use the default CAs, or False for no verification.",
     )
+    vault_path_prefix: str = Field(
+        ...,
+        description="Path prefix without leading or trailing slashes where secrets should"
+        + " be stored in the vault.",
+    )
 
```

Comment:
The name "prefix" is a bit misleading, because we will probably use prefixes to separate the deployments, like `testing-ekss` or `staging-ekss`, and there is no path component after that "prefix" (the key is not considered part of the path).

So I would just call it "vault_path" instead, and also rename "prefix" elsewhere to "path".

Reply:
Implemented in bad11f8371609ad6f59acbe13611021bdf1a21f9.
Also changed VaultAdapter to store path internally as to not pass around the path everywhere.
Missed the other endpoints also having defaults for the path initially, this is now also changed.